services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list = None,
+    actors_ids: list = None
+) -> QuerySet | Movie:

```

Comment:
Correct it 
QuerySet[Movie]

Reply:
The comment regarding the return type annotations is not resolved, correct them